### PR TITLE
#1917 Include triggeredid in status.changed and .finished events

### LINKED ISSUE #1917: Remediation Service: Include 'triggerid' property in remediation.status.changed and remediation.finished events
The `remediation.status.changed` and `remediation.finished` events sent by the remediation service should contain the `triggerid` property introduced in [SPEC 31](https://github.com/keptn/spec/pull/31). The value of this property should be the eventID of the `remediation.triggered` event that triggered the remediation workflow.

### PR DESCRIPTION
**Pull request by bacherfl**
Closes #1917 
Signed-off-by: Florian Bacher <florian.bacher@dynatrace.com>

### CODE REVIEW

**Review by johannes-b — DISMISSED** [review]
lgtm

File: remediation-service/handler/handler.go [hunk 1]
```diff
@@ -363,6 +363,18 @@ func (r *Remediation) sendEvaluationTriggeredEvent() error {
...
 
+	triggeredID := ""
+	remediations, err := getRemediationsByContext(r.Keptn.KeptnContext, r.Keptn.Event)
+	if err != nil {
+		r.Keptn.Logger.Error("could not retrieve open remediation: " + err.Error())
+	}
+
+	for _, remediation := range remediations {
+		if remediation.Type == keptn.RemediationTriggeredEventType {
+			triggeredID = remediation.EventID
```
Comment (christian-kreuzberger-dtx) [inline remediation-service/handler/handler.go:374]: should we "break" out of the loop once we have found the ID?

File: remediation-service/handler/handler.go [hunk 2]
```diff
@@ -433,6 +448,18 @@ func (r *Remediation) sendActionTriggeredEvent(actionTriggeredEventData keptn.Ac
...
```
Comment (christian-kreuzberger-dtx) [inline remediation-service/handler/handler.go:459]: should we "break" out of the loop once we have found the ID?

Comment (bacherfl) [reply remediation-service/handler/handler.go:459]: good point, I have added the break

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**